flink/v1.20/flink/src/test/java/org/apache/iceberg/flink/TestFlinkCatalogTable.java

```diff
@@ -188,6 +188,23 @@ public void testCreateTableLike() throws TableNotExistException {
         .isEqualTo(TableSchema.builder().field("id", DataTypes.BIGINT()).build());
   }
 
+  @TestTemplate
+  public void testCreateTableLikeInFlinkCatalog() throws TableNotExistException {
```

Comment:
Is there a simple way to check whether the create table like command is working even when the source table is not using the default catalog.

I'm a bit concerned that this test would pass if we simply use the default catalog for the new table instead copying the values from the source table.

Reply:
Don't understand this completely. We are testing when create table <iceberg_catalog> OR <Flink catalog>  LIKE <icebergtable>  . Source table is always iceberg table here. 
Let me know, if you still see this as issue after our offline discussion.

Reply:
NAh.. this is fine